Tests/Model/SPITransactionTests.m

```diff
@@ -141,6 +166,15 @@ - (void)testInitiateMotoPurchaseRequest {
     }];
 }
 
+- (void)testInitiateMotoPurchaseRequestWithSurchargeAmount {
+    SPIClient *client = [SPITestUtils clientWithTestSecrets];
+    
+    client.state.status = SPIStatusPairedConnected;
+    [client initiateMotoPurchaseTx:@"test" amountCents:19 surchargeAmount:1 completion:^(SPIInitiateTxResult *result) {
+        XCTAssertNotNil(result);
```

Comment:
Async unit tests in ios need to use XCTestExpectation or you can get all sorts of weirdness, like a subsequent test failing when this test was the culprit, see here for how to do it: https://developer.apple.com/documentation/xctest/asynchronous_tests_and_expectations/testing_asynchronous_operations_with_expectations

Reply:
This test is about Transactions via websocket. Previous test methods were developed like that.